shenyu-client/shenyu-client-motan/src/main/java/org/apache/shenyu/client/motan/common/annotation/ShenyuMotanService.java

```diff
@@ -33,6 +33,7 @@
 @Target({ElementType.TYPE, ElementType.METHOD})
```

Comment:
`@MotanService` only supports annotated in `ElementType.TYPE`, so a smaller `@Target` set should be selected.